modules/mono/csharp_script.cpp

```diff
@@ -103,13 +102,6 @@ void CSharpLanguage::init() {
 	}
 #endif
 
-#if defined(TOOLS_ENABLED) && defined(DEBUG_METHODS_ENABLED)
-	// Generate the bindings here, before loading assemblies. The Godot assemblies
-	// may be missing if the glue wasn't generated yet in order to build them.
```

Comment:
Would like @neikeq to confirm that it's fine if we now generate the bindings after loading the assemblies.
It seems to me like it's two different use cases which shouldn't conflict (editor generating the glue on one side, project assemblies being loaded on the other side), but I don't know what was the original problem that lead to this comment.

Reply:
The comment seems very outdated, but I'll review the blame to see what it was about.